source/_patterns/03-organisms/global/header/header.twig

```diff
@@ -8,10 +8,6 @@
         decorative: logo.decorative
       } %}
     </div>
-    {% include '@molecules/navigation/header-nav/header-nav.twig' with {
-      header_nav: {
-        items: menu_items
-      }
-    } %}
+    {% include '@molecules/navigation/header-nav/header-nav.twig' %}
```

Comment:
I think this accidentally reverts a change I made in the same commit as the full-height hero changes mentioned above. Seems like maybe that commit got somehow reverted in this merge.